ddtrace/tracer/option.go

```diff
@@ -829,3 +829,42 @@ func StackFrames(n, skip uint) FinishOption {
 		cfg.SkipStackFrames = skip
 	}
 }
+
+// UserTagOption represents a function that can be provided as a parameter to SetUser.
+// This is used to normalize user tags key/value pairs.
+type UserTagOption func() (key string, val interface{})
```

Comment:
I would rather call it `UserOption` or `UserMonitoringOption` trying to reflect what it does rather than how it does it.

Reply:
I'd prefer to go with `UserMonitoringOption`, my opinion is that `UserOption` is pretty vague